yarn.lock

```diff
@@ -2805,9 +2805,9 @@
     process-es6 "^0.11.2"
 
 "@stream-io/stream-chat-css@^2.2.1":
-  version "2.2.1"
-  resolved "https://registry.yarnpkg.com/@stream-io/stream-chat-css/-/stream-chat-css-2.2.1.tgz#23ce6ece96c2118340144191afe834234fb3f5d1"
-  integrity sha512-UJ/UM3wHCd3s1gqPIx2j2Vwpeoa1bbvISEarOlbioLphyHz7LC+E0+Hi81fSKMC9/mCa1ghugvolRKJgj3wwHQ==
+  version "2.8.0"
```

Comment:
If this is necessary for the fix, then you should bump explicitly the version in package.json. the yarn.lock has no effect on the package resolution for the package consumers.

Reply:
@petyosi fixed